azurelinuxagent/common/osutil/default.py

```diff
@@ -92,6 +94,7 @@ def __init__(self):
         self.agent_conf_file_path = '/etc/waagent.conf'
         self.selinux = None
         self.disable_route_warning = False
+        self.jit_enabled = True
```

Comment:
If you are setting this here, why are you also setting it in the concrete classes?

Reply:
I think this is leftover from an earlier bug... I thought some distros just defaulted to DefaultOSUtil but I was actually just missing setting this in a few places.

---
In reply to: [192192256](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r192192256) [](ancestors = 192192256)